e2e_playwright/conftest.py

```diff
@@ -124,6 +132,11 @@ def resolve_test_to_script(test_module: ModuleType) -> str:
     return test_module.__file__.replace("_test.py", ".py")
 
 
+def get_iframe_html_path() -> str:
+    """Get the absolute path of the given path."""
+    return f"file://{os.getcwd()}/test_assets/iframed_app.html"
```

Comment:
I think it's more stable / versatile to resolve the test_assets path via this: 

https://github.com/streamlit/streamlit/blob/82c9d3e7c6e44eb9dd6f093e200b049cce825652/e2e_playwright/st_video.py#L24

Reply:
Also this way, it doesn't need to be a method, it can just be a constant

Reply:
Ah good point, but actually we don't need the function anymore as we return the code inline and not from a file

Reply:
Oh I see 👍

Reply:
I removed it 🙂